package.json

```diff
@@ -22,7 +22,8 @@
   "dependencies": {
     "debug": "4.1.1",
     "got": "10.7.0",
-    "local-web-server": "^4.2.1"
+    "local-web-server": "^4.2.1",
+    "puppeteer": "^7.0.1"
```

Comment:
This version requires `node >=10.18.1` : https://github.com/puppeteer/puppeteer/blob/4e8d074c2f8384a2f283f5edf9ef69c40bd8464f/package.json#L8

Reply:
Also probably not the best approach to get a pinned version of a Chromium based browser